### DISCUSSION TITLE
Create a RBM with local connexions

### DISCUSSION THREAD

**Discussion by mgallegoq** [netket/netket#1464] [Q&A]
I am trying to define a RBM model that only has connexions between the first n-neighbours, i.e the visible node j has connexions with the hidden nodes from j-n to j+n. The first thing that comes to mind is to constraint the weights so W_{ij}=0 if |i-j|>n. Can this be done in netket?

**Comment by PhilipVinc** [comment]
Yes, you should use a convolution layer instead of a dense layer.

Check out the layers in flax
